src/util/std_types.cpp

```diff
@@ -1,11 +1,15 @@
 /*******************************************************************\
 
-Module:
+Module: Pre-defined types
 
 Author: Daniel Kroening, kroening@kroening.com
+        Maria Svorenova, maria.svorenova@diffblue.com
```

Comment:
I don't know, but should the additional Author be Diffblue @peterschrammel?

Reply:
I followed @peterschrammel recommendation from Slack but happy to change it to whatever makes more sense.